### ISSUE TITLE
Clarifications about context size

### ISSUE THREAD

**Issue by gonzalobenegas** [instadeepai/nucleotide-transformer#7]
Could you clarify what's the recommended context size for the models?
* The default  `max_positions` is 1024.
* The paper mentions maximum of 1000 tokens.
* The paper mentions 6kb sequences, would that mean 1000 6-mer tokens + 1 CLS token = 1001 tokens?

Thank you!

**Comment by dallatt** [comment]
Hello Gonzalo,

Thank you for your interest in our work!

I just added some clarification about how the tokenizer works and what it induces for the context length that can be used with these transformers. I also modified the hyperparameters as they were not truly reflective of the context length that was actually used to train these models.

You will now find that the `max_positions` is 1000 for all models, which corresponds to the 1000 tokens mentioned in the paper. Also, as now detailed in the [Tokenization](https://github.com/instadeepai/nucleotide-transformer#tokenization-abc) section of the README.txt, the 1000 tokens include the \<CLS> token.

Hope that answers everything,

Do not hesitate if you have any other questions.
Best,
Hugo
